text/profiles/0239-profiles-data-model.md

```diff
@@ -1349,7 +1349,7 @@ sample:
     locations_length: 3
     value:
       - 100
-  - locations_start_index: 2
+  - locations_start_index: 4
```

Comment:
Ah, my bad, thanks for fixing this, but I think it should be `3` actually:
```suggestion
  - locations_start_index: 3
```

And also, I think `location_indices` should be slightly different, there's a off by 1 error right now:
```
location_indices:
  - 0 # foo
  - 1 # bar
  - 2 # baz
  - 3 # abc
  - 4 # def
```

@florianl Is this right or am I missing something?

Reply:
sorry for the delay - you are right :sweat_smile: